handbooks/learners-meetups.md

```diff
@@ -25,6 +25,15 @@ Plus, a couple of things should be on the wall during the break:
 3. Learn'n'Tell session.
 Learn'n'Tell is inspired by [Hack'n'Tell](http://hackandtell.org/) (taking place in New York and Berlin), an event during which hackers can present their projects and get feedback from the audience. During our Learn'n'Tell session, beginners have 5 minutes to ask tech questions or show what they're working on (live coding is more than welcome!) and 5 minutes to receive feedback from the audience (experienced developers also come to the meetup to support the discussion). Important: have someone moderate the session and keep an eye on the time!
 
+OR
+
+Mini barcamp
+
+A barcamp is an "unconference", where the participants come up with the sessions themselves. There is a brainstorming phase where all participants write down on a sticky a question they have or topic they are interested in learning or talking about. Sticky notes are collected and stuck on the wall to form a "question cloud". The host reads out each question and asks for clarification if necessary. The host, with the help of the group, categorizes the stickies into related topics. Then the host calls out each topic group and asks who is interested in talking about it. The people who put up there hands form an informal discussion group.
+
+If people want to be involved in multiple groups, they can join at the half way mark. After 20 mins, give people a reminder that they can change groups if they are interested in other topics. To close (after 40 mins or earlier depending on the activity of the groups), bring everyone back together in a circle and nominate someone from each group to summarize what was discussed or learned.
```

Comment:
Good explanation! 

But I wonder about the phrase `and nominate someone from each group`. Maybe this could be rephrased to be a bit more collaborative e.g. `ask each group to share their learnings via a representative`. What do you think? Is this too cautious?